### ISSUE TITLE
Remove this repo, and migrate to `chain:address` format

### ISSUE THREAD

**Issue by shawntabrizi** [paritytech/ss58-registry#221]
Context: https://forum.polkadot.network/t/ux-proposal-consensus-based-address-formats/6072

Accepted Solutions:

- https://ethereum-magicians.org/t/chain-specific-addresses/6449

cc @joepetrowski

**Comment by bkchr** [comment]
Maybe should be a RFC?

**Comment by joepetrowski** [comment]
We still need a registry somewhere that gives the prefix for each consensus system, but then we would really just have three: Polkadot, Kusama, and testnets. So perhaps a repo/registry is overkill and we just go to using chain specs? In which case, I think @bkchr is right, we should have an RFC telling UI/app devs to use the prefix of the consensus system to which it belongs.

**Comment by bkchr** [comment]
I mean I'm not against removing/moving this repo to somewhere else. :)

**Comment by joepetrowski** [comment]
> Couldn't that live on-chain on the consensus system? Just query Kusama if you want to know the prefix.

Yes, it does, but that doesn't mean people look for it there or use it. I would say archive this repo, but we should also not just make it disappear but rather provide instructions about what to do instead of using it.
